.rubocop.yml

```diff
@@ -382,9 +361,6 @@ Style/SymbolArray:
 Style/IfUnlessModifier:
   Enabled: false
 
-Metrics/LineLength:
-  Enabled: false
-
```

Comment:
alright @shaigy - updated to remove this one instead!

Reply:
aaaand now it's mad because some of your files are too long 😁

Reply:
the options would be
1. switch back to disabling this cop
2. disable the cop only on the offending methods
3. increase the allowed line length